flow/layers/physical_shape_layer.h

```diff
@@ -9,41 +9,49 @@
 
 namespace flutter {
 
-class PhysicalShapeLayer : public ContainerLayer {
+#if defined(OS_FUCHSIA)
+using PhysicalShapeLayerBase = SystemCompositedContainerLayer;
+#else
+class PhysicalShapeLayerBase : public ContainerLayer {
  public:
-  PhysicalShapeLayer(SkColor color,
-                     SkColor shadow_color,
-                     float elevation,
-                     const SkPath& path,
-                     Clip clip_behavior);
-  ~PhysicalShapeLayer() override;
+  static bool should_system_composite() { return false; }
+
+  PhysicalShapeLayerBase(SkColor frame_color, float elevation)
+    : color_(frame_color),
+      elevation_(elevation) {}
+
+  SkColor color() const { return color_; }
+  float elevation() const { return elevation_; }
+
+ private:
+  SkColor color_;
+  float elevation_;
+};
+#endif
 
+class PhysicalShapeLayer : public PhysicalShapeLayerBase {
+ public:
   static void DrawShadow(SkCanvas* canvas,
                          const SkPath& path,
                          SkColor color,
                          float elevation,
                          bool transparentOccluder,
                          SkScalar dpr);
 
-  void Preroll(PrerollContext* context, const SkMatrix& matrix) override;
+  PhysicalShapeLayer(SkColor color,
```

Comment:
Why are we changing the order of the declarations here? It's a little harder to tell what's changed and what's not changed.

Reply:
I was trying to group similar declarations together.  Split this out into its own change to make it clearer what is going on